daemon/api.go

```diff
@@ -2427,7 +2446,8 @@ func postApps(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 	st.Lock()
 	defer st.Unlock()
-	chg := newChange(st, "service-control", fmt.Sprintf("Running service command"), tss, inst.Names)
+	// names can be snap or snap.app
```

Comment:
maybe mention that now all names will be just the snap name, but in previous versions could have been snap or snap.app?

Reply:
Hmm, but they can still be snap or snap.app. The bug was in storing the name as it was received in `snap-names`, so we ended up with `lxd.daemon` in the list, what was later interpreted as a snap name. Perhaps I should clarify the comment.

Reply:
how can it be snap.app now? It seems that `namesToSnapNames` is implemented such that only the snapName is put into seen, thus only the snapName is put into names and stored through this? Is there some other task/change that is storing as snap.app ?

Reply:
I mean you can still issue `snap restart lxd.daemon`, so inst.Names can still be snap or snap.app.

Reply:
makes sense now, thank you for adjusting the comment